sourcegraph/templates/prometheus/prometheus.RoleBinding.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if and .Values.prometheus.enabled (not .Values.prometheus.privileged) -}}
```

Comment:
I was reading through flow-control for helm templates and I could _NOT_ see any negation! Good to know!